src/components/StyledLink/index.tsx

```diff
@@ -13,6 +14,7 @@ export type StyledLinkProps = {
   icon?: boolean
   openNewTab?: boolean
   className?: string
+  download?: string
```

Comment:
This design may cause the component to carry too much functionality in the future, so it may be considered to change it to `rear?: ReactNode` or `render?: (content: ReactNode) => ReactNode` .

Reply:
`download` is a standard attribute of a link to set the file name, I may forget to append it to the `a` tag

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download

Reply:
Added by https://github.com/Magickbase/nervos-official-website/pull/359/commits/173349c88ecc8730316eac38cbc88ed2e207d3a8

Reply:
Oh, then its existence here seems very reasonable.